src/vdaf/prg.rs

```diff
@@ -0,0 +1,232 @@
+// SPDX-License-Identifier: MPL-2.0
+
+//! This module implements PRGs as specified in draft-patton-cfrg-vdaf-01.
+
+use crate::vdaf::{CodecError, Decode, Encode};
+use aes::{
+    cipher::{
+        generic_array::GenericArray, FromBlockCipher, NewBlockCipher,
+        StreamCipher as AesStreamCipher,
+    },
+    Aes128, Aes128Ctr,
+};
+use cmac::{Cmac, Mac, NewMac};
+use std::{
+    fmt::Debug,
+    io::{Cursor, Read},
+};
+
+/// Input of [`Prg`].
+#[derive(Clone, Debug, Eq)]
```

Comment:
nit: I think deriving `Copy` would also be appropriate.

Reply:
I think forcing the user to clone is better for this particular type. Happy to elaborate if you're interested.